Tools/autotest/autotest.py

```diff
@@ -609,17 +609,37 @@ def run_tests(steps):
 
     return passed
 
-def list_subtests(*args, **kwargs):
-    for vehicle in sorted(['Sub', 'Copter', 'Plane', 'Tracker', 'Rover', 'QuadPlane', 'BalanceBot', 'Helicopter']):
+
+vehicle_list = ['Sub', 'Copter', 'Plane', 'Tracker', 'Rover', 'QuadPlane', 'BalanceBot', 'Helicopter']
+
+
+def list_subtests():
+    """Print the list of tests and tests description for each vehicle."""
+    for vehicle in sorted(vehicle_list):
         tester_class = tester_class_map["test.%s" % vehicle]
         tester = tester_class("/bin/true", None)
         subtests = tester.tests()
         print("%s:" % vehicle)
-        for subtest in sorted(subtests, key=lambda x : x[0]):
+        for subtest in sorted(subtests, key=lambda x: x[0]):
             (name, description, function) = subtest
             print("    %s: %s" % (name, description))
         print("")
 
+
+def list_subtests_v(vehicle_type):
```

Comment:
Bytes are cheap here - could have spelt this out as `list_subtests_for_vehicle`

Reply:
sure!